documentation/operators/src/changelog.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+This page displays a full list of all the changes during our release cycle from [`v2024.3-eclipse`](https://github.com/nymtech/nym/blob/nym-binaries-v2024.3-eclipse/CHANGELOG.md) onwards. Operators can find here the newest updates together with links to relevant documentation. The list is sorted in a way that the most recent changes are at the top.
+
+## `v2024.3-eclipse`
+
+- Release [Changelog.md](https://github.com/nymtech/nym/blob/nym-binaries-v2024.3-eclipse/CHANGELOG.md)
+- The [`nym-node`](nodes/nym-node.md) initial release
```

Comment:
nym-node initial release, in the film the social network he said drop the "the" just "nym-node

Reply:
true - done